style.css

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
You ended up using a lot of the same attributes for different tags. Consider adding a class to them in the future, then you won't have to repeat yourself as much